mchf-eclipse/drivers/audio/cw/cw_gen.c

```diff
@@ -226,7 +226,7 @@ static const float sm_table[CW_SMOOTH_TBL_SIZE] =
 
 void CwGen_SetSpeed()
 {
-    ps.dit_time         = 1650/ts.keyer_speed;      //100;
+    ps.dit_time         = 1920/ts.keyer_speed;      //100;
 }
```

Comment:
This formula for calculating key on time is probably correct (I still have to test it):

ps.dit_time = 1800/ts.keyer_speed+9; // +9 = 6ms / (1/1500) = 0,006*1500

The 6 ms is the rise and fall time combined of the CW smoothing.
I will test it tonight (I'm not near my mcHF now). I will create a new formula to calculate the pause time. 
(A wild guess would be: pause_time = 1800/ts.keyer_speed-9; )